frontend/src/app/general/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,33 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="dialog" id="custom-dialog">
+  <h2>{{ data.title }}</h2>
+  <p>{{ data.text }}</p>
+  <p *ngIf="data.requiredInput">
+    Please type in <span class="font-bold">{{ data.requiredInput }}</span> to
+    confirm the action:
+  </p>
+  <mat-form-field
+    class="w-full"
+    appearance="outline"
+    *ngIf="data.requiredInput"
+  >
+    <input matInput [(ngModel)]="inputText" placeholder="Type here..." />
+  </mat-form-field>
+  <div class="flex justify-between">
+    <button mat-flat-button color="primary" (click)="onCancelClick()">
+      Cancel
+    </button>
+    <button
```

Comment:
Submit on enter doesn't work.

Reply:
Wasn't sure if we do want to have this for such a confirmation dialog as explicitly clicking on "confirm" is somehow another safety mechanism but I its now added.